Add step to create Python file and modify svg for interpreter selection

luabud opened the pull request:
Closes https://github.com/microsoft/vscode-python/issues/18287

The svg modification is just temporarily until we decide where we will end up placing the Python info on the status bar

luabud commented:
JFYI @misolori -- I edited (and potentially butchered) one of the SVGs but that's just temporarily 😅 we are going to try out moving the status bar item to the right in an experiment (more info here: https://github.com/microsoft/vscode-python/issues/18283), so I figured it was worth hiding the status bar from the interpreter selection SVG in the mean time to not confuse users.

kimadeline commented on package.json:
```diff
@@ -132,20 +132,30 @@
+                    {
+                        "id": "python.createPythonFile",
+                        "title": "Create a Python file",
+                        "description": "Create a new file and save it as \".py\" to start with your Python development. \n[Create Python File](command:toSide:workbench.action.files.newUntitledFile)",
```
```suggestion
                        "description": "Create a new file and save it as \".py\" to get started. \n[Create Python File](command:toSide:workbench.action.files.newUntitledFile)",
```

kimadeline commented on package.json:
```diff
@@ -132,20 +132,30 @@
                         "id": "python.runAndDebug",
                         "title": "Run and debug your Python file",
-                        "description": "[Open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.visualstudio.com/docs/pyth ...
+                        "description": "If you haven't already, [open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.vi ...
```
I don't think this change adds anything to the step. If you want to keep it, I would suggest the following:
```suggestion
                        "description": "If you haven't done so already, [open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:welcome.showNewFileEntries) a Python file - make sure to save it as \".py\". \n Click on the play button on the top right of the editor, or press F5 when on the file and select \"Python File\" to run with the debugger. \n  \n[Learn more](https://code.visualstudio.com/docs/python/python-tutorial#_run-hello-world)",
```

kimadeline reviewed (commented):
Do you have a screenshot of what the new SVG looks like?

luabud commented:
### Screenshots 

Light theme:
![image](https://user-images.githubusercontent.com/45497113/149428445-4cfaa2a6-1da7-4355-9d7d-a96c1d9b3182.png)


Dark theme: 
![image](https://user-images.githubusercontent.com/45497113/149428480-65b63130-430d-4ae3-9a69-2b528545b6ee.png)

luabud commented:
@kimadeline thank you so much for the review! I agree with you regarding the "if you haven't done so" bit -- it was unnecessary so I just removed it 😁

karrtikr commented on package.json:
```diff
@@ -132,12 +132,22 @@
+                    },
                     {
                         "id": "python.selectInterpreter",
                         "title": "Select a Python Interpreter",
                         "description": "Choose which Python interpreter/environment you want to use for your Python project.\n[Select Python Interpreter](command:python.setInterpreter)\nTip: Reload the window if you installed Python but don't see it in the list (``Developer: Reload Window`` command in the [Command Palette](command:workbench.action.showCommands))",
```
Does it make sense to mention the `Python: Select Interpreter` command here somewhere now that we have cropped the original screenshot?

luabud commented:
Thanks for the review and feedback @karrtikr! I added a tip to include the select interpreter command,  and removed the "open or create" actions/links from the "run and debug" tile, so now they are just in the "create file" step. I ended up not using the `pythonInterpreterInfoUnpinned` context though, because I figured that could be "flickerish" situations where the tile would show up or disappear when the flight information is updated 🤔

luabud commented:
![image](https://user-images.githubusercontent.com/45497113/149846079-090133f4-5248-4fe8-bf92-a87e7b5e6c97.png)

![image](https://user-images.githubusercontent.com/45497113/149846094-a0cab168-e166-40ba-8bea-5feb76e41e33.png)

![image](https://user-images.githubusercontent.com/45497113/149846104-8e9a587e-4159-48b4-9564-7e105941bbb8.png)

![image](https://user-images.githubusercontent.com/45497113/149846114-7304ace6-a5e1-4871-bb5f-5d70d1c9f5ff.png)

![image](https://user-images.githubusercontent.com/45497113/149846132-83c4c67b-04df-4f23-ab79-d48993517162.png)

karrtikr reviewed (approved):
LGTM
